cfg/defaults.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Function that returns default values.
+ * Used because Object.assign does a shallow instead of a deep copy.
+ * Using [].push will add to the base array, so a require will alter
+ * the base array output.
+ */
+'use strict';
+
+const path = require('path');
+const srcPath = path.join(__dirname, '/../src');
+const dfltPort = 8000;
+
+/**
+ * Get the default modules object for webpack
+ * @return {Object}
+ */
+function getDefaultModules() {
+  return {
+    preLoaders: [
+      {
+        test: /\.(js|jsx)$/,
+        include: srcPath,
+        loader: 'eslint-loader'
+      }
+    ],
+    loaders: [
+      {
+        test: /\.css$/,
+        loader: 'style-loader!css-loader'
+      },
+      {
+        test: /\.sass/,
+        loader: 'style-loader!css-loader!sass-loader?outputStyle=expanded&indentedSyntax'
+      },
+      {
+        test: /\.scss/,
+        loader: 'style-loader!css-loader!sass-loader?outputStyle=expanded'
+      },
+      {
+        test: /\.less/,
```

Comment:
Lets delete the loaders that we're not using... I think .sass, .scss, .styl, and .less can all be removed